test/e2e/templates/zone-wait/pod.yaml

```diff
@@ -0,0 +1,200 @@
+---
```

Comment:
Do we need to create individual PVC and pods to consume them, or instead use a statefulset like what Luke achieved and dynamically create them and consume them?

Reply:
I can convert it to the statefulset, I just have to rework the checks to ensure that everything is up and running for the pods. Will take a look.